.github/workflows/build.yml

```diff
@@ -34,8 +34,7 @@ jobs:
     - name: Build pbstate
       run: |
             pushd tools/pbstate
-            autoreconf -fi
-            ./configure
+            cmake .
```

Comment:
CMake generally don't like at all to build in same directory (which is probably why CI fails, I haven't checked).